cdap-app-fabric/src/main/java/co/cask/cdap/metadata/DefaultMetadataAdmin.java

```diff
@@ -216,10 +216,10 @@ private void validateKeyAndTagsFormat(MetadataEntity metadataEntity, String keyw
   private void validateValueFormat(MetadataEntity metadataEntity, String keyword)
     throws InvalidMetadataException {
     if (!VALUE_MATCHER.matchesAllOf(keyword)) {
-      throw new InvalidMetadataException(metadataEntity, String.format("Illegal format for the value '%s'. Must only " +
-                                                                         "consists of characters[a-z or A-Z], " +
-                                                                         "digits[0-9], underscore(_), hyphen(-) or " +
-                                                                         "whitespace.", keyword));
+      throw new InvalidMetadataException(metadataEntity, String.format("Illegal format for the value '%s'. Should " +
```

Comment:
should we remove value from here too? can this ever be a tag?

Reply:
This is the value of the key-value property. This cannot be a tag. Tag needs to follow the pattern of KEY_AND_TAG_MATCHER